src/views/payment-review.njk

```diff
@@ -16,15 +16,17 @@
           {% set paymentRows =
             (
               paymentRows.push(
```

Comment:
Realised there was an issue when working on the "Application status" table, which meant that multiple objects were added onto one long row (rather than separate multiple rows).
The array `[]` needed added to the `paymentRows` variable (rather than to the `rows` property on the `govukTable`).

Can see the documentation on how tables work on the GDS design system here, https://design-system.service.gov.uk/components/table/